src/app/components/HeaderBar/HeaderBar.module.css

```diff
@@ -11,6 +11,22 @@
 .catLogo {
   grid-column: 1;
   padding: 2em 0 0 1em;
+  animation: wobble 1s 1;
+}
+
+@keyframes wobble {
+  25% {
+    transform: rotate(15deg);
+  }
+  50% {
+    transform: rotate(-30deg);
+  }
+  75% {
+    transform: rotate(5deg);
+  }
+  100% {
+    transform: rotate(0deg);
+  }
 }
```

Comment:
You have the same animation multiple times in your app. Consider outsourcing it or adding it to your globals.
https://github.com/css-modules/css-modules#composing-from-other-files